tests/link/pytorch/test_basic.py

```diff
@@ -301,3 +302,20 @@ def test_pytorch_MakeVector():
     x_fg = FunctionGraph([], [x])
 
     compare_pytorch_and_py(x_fg, [])
+
+
+def test_pytorch_ifelse():
+    true_vals = np.r_[1, 2, 3]
+    false_vals = np.r_[-1, -2, -3]
+
+    x = ifelse(np.array(True), true_vals, false_vals)
```

Comment:
No point in testing this since everything is constant the dispatch won't even be triggered?

Reply:
That sounds good - this was copy-pasta from the jax test